packages/calcite-components/src/components/filter/filter.e2e.ts

```diff
@@ -67,6 +67,15 @@ describe("calcite-filter", () => {
     expect(await input.getProperty("scale")).toBe(scale);
   });
 
+  it("honors label property", async () => {
+    const page = await newE2EPage();
+    const label = "hell world";
```

Comment:
😨! Did you mean to type hello 👋🏻?

Reply:
whoops lol

Reply:
![](https://media.giphy.com/media/dvkFZr4VBBS6I/giphy.gif?cid=790b76111lfi8bw02bs8ujv4zi2hpmshz5khq960ufpewfyd&ep=v1_gifs_search&rid=giphy.gif&ct=g)